x-pack/platform/plugins/shared/index_management/public/application/services/breadcrumbs.ts

```diff
@@ -194,6 +203,16 @@ class BreadcrumbService {
         href: `/enrich_policies/create`,
       },
     ];
+    this.breadcrumbs.indicesList = [
```

Comment:
This has the same content as `this.breadcrumbs.indices` but with an additional item prepended to the list, so it could be defined in terms of the existing breadcrumb.